src/views/forms/AccountManagementForms/TokenSelectMenu.tsx

```diff
@@ -24,6 +25,11 @@ type ElementProps = {
   isExchange?: boolean;
 };
 
+export const tokenIsCCTPToken = (token: SelectionOption): boolean => {
```

Comment:
[discussion] anyone have any opinions on creating a `utils/cctp` file and moving this there?